File: ComplementApp.API/Dtos/TerceroDeduccionDto.cs
namespace ComplementApp.API.Dtos
{
    public class TerceroDeduccionDto
    {
        public int TerceroDeduccionId { get; set; }
        public int TipoIdentificacion { get; set; }
        public string IdentificacionTercero { get; set; }
        public string Codigo { get; set; }        
        public ValorSeleccion Tercero { get; set; }
        public DeduccionDto Deduccion { get; set; }
        public ValorSeleccion ActividadEconomica { get; set; }
        public ValorSeleccion TerceroDeDeduccion { get; set; }
        public int EstadoModificacion { get; set; }
        public decimal ValorFijo { get; set; }
    }
}

File: ComplementApp-SPA/src/app/_dto/terceroDeduccionDto.ts
import { DeduccionDto } from './deduccionDto';
import { ValorSeleccion } from './valorSeleccion';

export interface TerceroDeduccionDto {
  terceroDeduccionId: number;
  tipoIdentificacion: number;
  identificacionTercero: string;
  codigo: string;
  tercero: ValorSeleccion;
  deduccion: DeduccionDto;
  actividadEconomica: ValorSeleccion;
  terceroDeDeduccion: ValorSeleccion;
  estadoModificacion: number;
  esValorFijo: boolean;
  valorFijo: number;
}
